fixtures/proc-macro/src/lib.rs

```diff
@@ -275,4 +273,28 @@ pub fn join(parts: &[String], sep: &str) -> String {
     parts.join(sep)
 }
 
+/// Struct with a bunch of default values, this is our main way to test defaults
+#[derive(uniffi::Record)]
+pub struct DefaultTester {
+    no_default_string: String,
+    #[uniffi(default = true)]
+    boolean: bool,
+    #[uniffi(default = 42)]
+    integer: i32,
+    #[uniffi(default = 4.2)]
+    float: f64,
+    #[uniffi(default=[])]
+    vec: Vec<bool>,
+    #[uniffi(default=None)]
+    opt_vec: Option<Vec<bool>>,
+    #[uniffi(default = Some(42))]
+    opt_integer: Option<i32>,
+}
+
+/// Also test that defaults work with functions
+#[uniffi::export]
+fn double_with_default(#[uniffi(default = 21)] num: i32) -> i32 {
```

Comment:
Looking at #1929, this should read `#[uniffi::export(default(num = 21))]`? Indeed, this is already landed. It looks like this PR needs updating?